typescript/infra/config/environments/testnet3/chains.ts

```diff
@@ -20,12 +22,21 @@ export const testnetConfigs: ChainMap<ChainMetadata> = {
 
 // "Blessed" chains that we want core contracts for.
 export type TestnetChains = keyof typeof testnetConfigs;
-export const chainNames = Object.keys(testnetConfigs) as TestnetChains[];
+export const supportedChainNames = Object.keys(
+  testnetConfigs,
+) as TestnetChains[];
 export const environment = 'testnet3';
 
-// Chains that we want to run agents for.
-export const agentChainNames = [
-  ...chainNames,
+const validatorChainNames = [
+  ...supportedChainNames,
   'solanadevnet',
```

Comment:
nit: I should've done this before, but can you change this to `chainMetadata.solanadevnet.name` and the same for `proteustestnet`